specification/network/resource-manager/Microsoft.Network/stable/2020-04-01/loadBalancer.json

```diff
@@ -549,9 +549,6 @@
         "x-ms-examples": {
           "Update load balancer backend pool with backend addresses containing virtual network and  IP address.": {
             "$ref": "./examples/LBBackendAddressPoolWithBackendAddressesPut.json"
```

Comment:
do the same change for 2020-05-01 folder.

Plus remove the example + its reference from put operation.